eml/constants/Fonts.js

```diff
@@ -3,6 +3,8 @@ import { useFonts } from 'expo-font';
 export const isFontsLoaded = () => {
   const [fontsLoaded] = useFonts({
     'Montserrat-Regular': require('../assets/fonts/Montserrat-Regular.ttf'),
```

Comment:
We should probably have all the fonts in here, and also make sure they are defined in tailwind